packages/core/content-releases/shared/contracts/release-actions.ts

```diff
@@ -64,7 +65,7 @@ export declare namespace GetReleaseActions {
   }
 
   export interface Response {
-    data: ReleaseAction & { entry: ReleaseActionEntryData }[];
+    data: (ReleaseAction & { entry: ReleaseActionEntryData })[];
```

Comment:
tip: this might have been easier doing `Array<ReleaseAction & { entry: ReleaseActionEntryData }>`